pkg/console/starter/starter.go

```diff
@@ -163,11 +167,12 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	clusterOperatorStatus := status.NewClusterOperatorStatusController(
 		"console",
 		[]configv1.ObjectReference{
-			{Group: "operator.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "config.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "oauth.openshift.io", Resource: "oauthclients", Name: api.OAuthClientName},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleOperatorNamespace},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleNamespace},
+			{Group: operatorv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "ingress", Name: api.ConfigResourceName},
```

Comment:
shouldn't thus be plural ?
```suggestion
			{Group: configv1.GroupName, Resource: "ingresses", Name: api.ConfigResourceName},
```

Reply:
Looks like you are correct:

https://github.com/openshift/cluster-config-operator/blob/master/manifests/0000_05_config-operator_01_ingress.crd.yaml#L10

I don't say `ingress` or `ingresses` often 😄